test/sign-verify.spec.js

```diff
@@ -30,6 +31,23 @@ describe('sign and verify', () => {
     result.should.be.true;
   });
 
+  it('has proper signature format', async () => {
+    const data = stringToUint8Array('test 1234');
+    const signature = await signer.sign({data});
+
+    // check some expected properties of a Uint8Array object
+    expect(signature).to.have.property('BYTES_PER_ELEMENT');
+    expect(signature).to.have.property('buffer');
+    expect(signature).to.have.property('byteLength');
+    expect(signature).to.have.property('byteOffset');
+    expect(signature).to.have.property('entries');
+    expect(signature).to.have.property('fill');
+    expect(signature).to.have.property('find');
+    expect(signature).to.have.property('includes');
+    expect(signature).to.have.property('slice');
+    expect(signature).to.have.property('sort');
```

Comment:
I think should just check `signature instanceof Uint8Array` because that's what consumers will be expecting:

```suggestion
    // check that signature is a `Uint8Array` instance
    expect(signature instanceof Uint8Array).to.be.true;
```

If there's some other chai/expect chaining that does that more elegantly like maybe `to.be.instanceof(Uint8Array)` or something, feel free to do that instead.

Reply:
Aha! I was trying to think of the right operator for this. I knew `typeof` wasn't going to work, but I didn't think of `instanceof`. Thanks!